networkdb/networkdb.go

```diff
@@ -285,7 +285,6 @@ func (nDB *NetworkDB) CreateEntry(tname, nid, key string, value []byte) error {
 	nDB.indexes[byNetwork].Insert(fmt.Sprintf("/%s/%s/%s", nid, tname, key), entry)
 	nDB.Unlock()
 
-	nDB.broadcaster.Write(makeEvent(opCreate, tname, nid, key, value))
```

Comment:
am not against this change. But I like to understand this a bit more.

`broadcaster.Write` ultimately calls appropriate Watch routines such as `handleEpTableEvent` or `handleNodeTableEvent` or other tables managed by other components (such as overlay driver).

CreateEntry in networkdb is a generic API to create an event of any type. But changing this generic function for a specific case of Endpoint management seems risky without considering how it impacts other events.
What if there is another table (not endpoint), with an event created by one component and consumed not just by remote peers, but also by another local component which networkdb as the mechanism for communication ?

Also, if we remove this call, can you point me to another code path for which handleEpTableEvent is called for local endpoints ?